From 028ec9c7e9f7a6f083eec987f3ad7e7623398d9d Mon Sep 17 00:00:00 2001
From: Ovidiu Panait <ovidiu.panait@windriver.com>
Date: Mon, 13 Jul 2020 12:12:31 +0300
Subject: [PATCH] nss: fix incorrect shebang of perl

Replace incorrect shebang of perl with `#!/usr/bin/env perl'.

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
Upstream-Status: Pending
Signed-off-by: Ovidiu Panait <ovidiu.panait@windriver.com>

---
 nss/cmd/signver/examples/1/form.pl       | 2 +-
 nss/cmd/signver/examples/1/signedForm.pl | 2 +-
 nss/cmd/smimetools/smime                 | 2 +-
 nss/coreconf/version.pl                  | 2 +-
 nss/tests/clean_tbx                      | 2 +-
 nss/tests/iopr/server_scr/client.cgi     | 2 +-
 nss/tests/path_uniq                      | 2 +-
 7 files changed, 7 insertions(+), 7 deletions(-)

diff --git a/nss/cmd/signver/examples/1/form.pl b/nss/cmd/signver/examples/1/form.pl
index f2cfddc..af58d54 100755
--- a/nss/cmd/signver/examples/1/form.pl
+++ b/nss/cmd/signver/examples/1/form.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/perl
+#!/usr/bin/env perl
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
diff --git a/nss/cmd/signver/examples/1/signedForm.pl b/nss/cmd/signver/examples/1/signedForm.pl
index 847814c..64a31ff 100755
--- a/nss/cmd/signver/examples/1/signedForm.pl
+++ b/nss/cmd/signver/examples/1/signedForm.pl
@@ -1,4 +1,4 @@
-#! /usr/bin/perl
+#!/usr/bin/env perl
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
diff --git a/nss/cmd/smimetools/smime b/nss/cmd/smimetools/smime
index e67f6be..6cd85e6 100755
--- a/nss/cmd/smimetools/smime
+++ b/nss/cmd/smimetools/smime
@@ -1,4 +1,4 @@
-#!/usr/local/bin/perl
+#!/usr/bin/env perl
 
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
diff --git a/nss/coreconf/version.pl b/nss/coreconf/version.pl
index d2a4942..3ba7323 100644
--- a/nss/coreconf/version.pl
+++ b/nss/coreconf/version.pl
@@ -1,4 +1,4 @@
-#!/usr/sbin/perl
+#!/usr/bin/env perl
 #
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
diff --git a/nss/tests/clean_tbx b/nss/tests/clean_tbx
index 4de9555..c15a069 100755
--- a/nss/tests/clean_tbx
+++ b/nss/tests/clean_tbx
@@ -1,4 +1,4 @@
-#! /bin/perl
+#!/usr/bin/env perl
 
 #######################################################################
 #
diff --git a/nss/tests/iopr/server_scr/client.cgi b/nss/tests/iopr/server_scr/client.cgi
index 581ad06..34ea170 100644
--- a/nss/tests/iopr/server_scr/client.cgi
+++ b/nss/tests/iopr/server_scr/client.cgi
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/env perl
 
 # This Source Code Form is subject to the terms of the Mozilla Public
 # License, v. 2.0. If a copy of the MPL was not distributed with this
diff --git a/nss/tests/path_uniq b/nss/tests/path_uniq
index f29f60a..850332a 100755
--- a/nss/tests/path_uniq
+++ b/nss/tests/path_uniq
@@ -1,4 +1,4 @@
-#! /bin/perl
+#!/usr/bin/env perl
 
 ########################################################################
 #
